software.html

```diff
@@ -0,0 +1,6 @@
+---
+layout: default
+title: Open Source Software Developed by the Viswanathan Group
```

Comment:
Title is too long for mobile devices

Is all software open source? I.e. I'd like to link to Incepts here but, I can't under the heading OSS

That said, I'm all for adding OSS shields on each entry (Think: https://shields.io/category/license)

Idea: Could we add more shields? Think build status

Reply:
I think that closed source codes should at the very least be in a separate section. As someone navigating the website, it would be nice for me to see that I can't access the closed source codes, rather than navigating all around the incepts website trying to figure out how to access the GitHub.

Reply:
Agreed. I'd very much like to celebrate our OSS offerings.

Maybe something like:

‘’’
# OSS Software

- BattModels.jl
- other cools things

# Group Developed Software
- Incepts
- other closed source
‘’’

Or were you thinking of a separate page? I don't want to clutter the navbar.

That said if Incepts is the only closed source offering, I think we limit it to OSS only. Closed Source Tools might be better suited to getting project pages, than showing up under software.

Reply:
I don't know of any other closed source codes, but that layout is roughly what I was thinking.